ext/standard/browscap.c

```diff
@@ -758,6 +746,18 @@ PHP_FUNCTION(get_browser)
 		size_t cached_prev_len = 0; /* silence compiler warning */
 
 		ZEND_HASH_MAP_FOREACH_PTR(bdata->htab, entry) {
+			/* Agent name too short */
+			if (ZSTR_LEN(lookup_browser_name) < browscap_get_minimum_length(entry)) {
+				continue;
+			}
+
+			/* Quickly discard patterns where the prefix doesn't match. */
+			if (zend_binary_strcasecmp(
+					ZSTR_VAL(lookup_browser_name), entry->prefix_len,
+					ZSTR_VAL(entry->pattern), entry->prefix_len) != 0) {
+				continue;
+			}
```

Comment:
Add a comment explaning why this is not in the comparison function so people know why

Reply:
Right, good point. Otherwise someone else in the future may move them again.